src/ui/pages/settings/tabs/GeneralSettings.svelte

```diff
@@ -108,6 +109,35 @@
   {/snippet}
 </SettingItem>
 
+<SettingItem
+  name="Excluded Folders"
+  description="Folders to exclude from annotation collection (one per line)"
+  notices={[
+    {
+      type: "info",
+      text: "Files in these folders will not appear in the annotations view (e.g., <code>_templates/</code>)",
+    },
+    {
+      type: "warning",
+      text: "Changing this setting results in a re-indexation of the entire vault",
+    },
+  ]}
+>
+  {#snippet control()}
+    <TextArea
```

Comment:
I am not sure whether a regular `TextArea` is the most ergonomic way to input paths, but I'll need to think about this.